server/controllers/blog.js

```diff
@@ -0,0 +1,75 @@
+const { Op } = require('sequelize');
+const { Blog } = require('../models');
+
+//  getting all the blog post limited to first 10
+
+exports.getAllBlog = (req, res, next) => {
+  const count = 10;
```

Comment:
Question: Should this be constant or do we want the client to be able to specify how many posts to get?